### PR TITLE
feat: provide dockerfile

### PR DESCRIPTION
**Pull request by fjogeleit**
Provide Dockerfile and basic make command

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ARG LD_FLAGS="-s -w"
+ARG TARGETPLATFORM
+
+WORKDIR /app
+
+COPY backend .
+
+COPY --from=frontend /var/www/dist /app/dist
+
+RUN export GOOS=$(echo ${TARGETPLATFORM} | cut -d / -f1) && \
```
[the remark this answers is not in the document]
Comment (fjogeleit) [reply Dockerfile:25]: @sonatype-lift ignore

[the remark this answers is not in the document]
Comment (fjogeleit) [reply Dockerfile:25]: @sonatype-lift ignore

File: backend/main.go [hunk 2]
```diff
@@ -163,6 +163,7 @@ func (r apiRequest) process(ctx context.Context) (*apiResponse, error) {
 			store.ContextLoaderFactory(nil),
 			nil,
 		)
+		var err error
```
[the remark this answers is not in the document]
Comment (fjogeleit) [reply backend/main.go]: removed it, got a warning from my IDE, so I tested it this way.

**Comment by eddycharly** [comment]
@fjogeleit there's a conflict

**Review by eddycharly — APPROVED** [review]

**Pull request merged**